operatorcourier/validate.py

```diff
@@ -13,6 +13,30 @@ class ValidateCmd():
     crdKey = "customResourceDefinitions"
     csvKey = "clusterServiceVersions"
     pkgsKey = "packages"
+    validationJSON = dict(
+        warnings=[],
+        errors=[],
+    )
+
+    def _log_warning(self, message):
```

Comment:
Is there a better place where we can place the log json data and custom log methods, or even implement our custom logger module? Since the user specifies the argument in the command line, he may also expect to see logs from other modules like [here](https://github.com/operator-framework/operator-courier/pull/27/files#diff-5fea1be6dfcb1f0a399be5863239c7a5R49). The current implementation does not handle that.

Reply:
`custom logger module` what do you mean by this? Since this PR starts I'm fighting against messing with logging.
I need probably an example.

Reply:
The goal is to capture all validation warnings and errors. As @MartinBasti mentioned above, implementing our own logging module is overkill.

Reply:
@MartinBasti I think my point is that the logger methods currently live in `validate.py`, which doesn't seem right. We also need log info in files in other places like [push.py](https://github.com/operator-framework/operator-courier/blob/master/operatorcourier/push.py#L46).

Reply:
@jeremylinlin but right now we aren't creating this log for anything except the validation. If we decide to expand it later to other areas that might be a concern then. For now I think it is fine to leave it as is, not much more than a package import at this point

Reply:
What is your use case with that push.py? What kind of information you need?

I'm even skeptical with that output file.
* STDERR is used for logging and errors
* STDOUT should be used for application output

To get validation errors:
```bash
operator-courier verify path/to/archive --output-type=json | jq '.validation.errors'
```

Reply:
`operator-courier push` can provide output in format like:
```json
{
  "validation": {
    "errors": ["error1", "error2"],
    "warnings": [],
    "status": "failed"
  },
  "build": {
      "status": "cancelled"
   },
  "push": {
       "status": "canceled"
   } 
}
```

And it has nothing wit logging, that output must be constructed from separated parts in cli and reported to user. I cannot imagine how to achieve this with simple logging.